src/components/common/Detail.vue

```diff
@@ -50,9 +48,19 @@ export default class Detail extends Vue {
   @Action setDetailComment: ActionBindingIF
   @Action setDetailValidity: ActionBindingIF
 
+  /** Maximum length of comment field in db. */
+  readonly MAX_LENGTH = 4096
+
   private comment: string = null
-  private maxLength: number = 4038
-  private textAreaStyle: string='filled'
+  private textAreaStyle = 'filled'
+
+  private get detailLabel (): string {
+    return `Correction for Incorporation Application. Filed on ${this.getOriginalIAFilingDate}`
+  }
+
+  private get maxLength (): number {
+    return (this.MAX_LENGTH - this.detailLabel.length)
```

Comment:
Since the detailLabel is not passed to the BE as of now, why would the max length take into consideration the length of that text

Reply:
Oh, it's supposed to. See:
https://github.com/bcgov/business-filings-ui/blob/69cd05f8f7265bdd2895560813dbde0902228447/src/views/Correction.vue#L68
https://github.com/bcgov/business-filings-ui/blob/69cd05f8f7265bdd2895560813dbde0902228447/src/views/Correction.vue#L304

Do we have a ticket for that?

Reply:
I dont think so

Reply:
Now we do: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5203

I will convert this PR to a draft until the bug is triaged (and rewiewed by PO/BAs) ... in case requirements change.

Reply:
Sounds good